tasks/fine_tuning_sbert/notebooks/Fine_tuning_SBERT_with_callback.ipynb

```diff
@@ -1,3160 +1,2839 @@
+            "text/plain": [
+              "<Figure size 432x288 with 1 Axes>"
+            ]
+          },
+          "metadata": {
+            "tags": [],
+            "needs_background": "light"
+          }
+        },
+        {
+          "output_type": "stream",
+          "text": [
+            "Label counts:\n",
+            "{'Incentive': 0.49146757679180886, 'not_Incentive': 0.5085324232081911}\n"
+          ],
+          "name": "stdout"
+        }
       ]
-     },
-     "metadata": {
-      "tags": []
-     },
-     "output_type": "display_data"
     },
     {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "============================================================\n",
-      "Current Score is: 0.9382716049382716\n",
-      "Current ACC_LIST is: [0.8148148148148148, 0.9444444444444444, 0.9382716049382716]\n",
-      "Epoch: 2\n",
-      "Best score: 0.9444444444444444\n",
-      "============================================================\n"
-     ]
+      "cell_type": "code",
+      "metadata": {
+        "id": "v1MTStwySrYL"
+      },
+      "source": [
+        "# PASTE YOUR WEIGHTS & BIASES KEY HERE\n",
+        "wandb_key = ''\n",
+        "group_desc = ''\n",
+        "job_type = ''\n",
+        "\n",
+        "os.environ['WANDB_JOB_TYPE'] = job_type\n",
+        "os.environ['WANDB_RUN_GROUP'] = group_desc\n",
+        "os.environ['WANDB_API_KEY'] = wandb_key"
+      ],
+      "execution_count": 7,
+      "outputs": []
     },
     {
-     "data": {
-      "application/vnd.jupyter.widget-view+json": {
-       "model_id": "72a901b909e144dc895a3d038d6083de",
-       "version_major": 2,
-       "version_minor": 0
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "VbnqVcEzuWq5"
       },
-      "text/plain": [
-       "HBox(children=(FloatProgress(value=0.0, description='Iteration', max=41.0, style=ProgressStyle(description_wid…"
+      "source": [
+        "## Grid Search Fine Tuning - W&B sweeps"
```

Comment:
In this section and the next one (run training function for one run), I'd add some explanation/instructions on how to use a "sweep" and which cells to run, that way there's a clear distinction and everyone knows which cells to run when, and how to look at the results.

Reply:
That's a good idea! Adding the comments now for clarity